src/main/webapp/js/actions/index.js

```diff
@@ -95,3 +99,47 @@ export default function (callback) {
     );
   };
 }
+
+export function fetchFieldConfig(callback) {
+  return function action(dispatch) {
+    const request = apiClient.get(`${FIELD_CONFIG}/VACCINEE`);
+    return request.then(
+      (response) => {
+        dispatch({
+          type: FETCH_VACCINEE_CONFIG,
+          payload: response,
+        });
+        callback();
+      },
+    );
+  };
+}
+
+export function saveFieldConfig(item) {
+  return function action(dispatch) {
```

Comment:
you don't need to use redux thunk here, you can just return the request in the payload, redux-promise will take care of it